### PR TITLE
docs(TaskProcessing): Update docs to reflect latest changes

### PR DESCRIPTION
**Pull request by marcelklehr**
### 🖼️ Screenshots

image

### CODE REVIEW

**Review by bigcat88 — APPROVED** [review]
Looks nice!

**Comment by marcelklehr** [comment]
(there are more changes than what is in the screenshot)

File: admin_manual/ai/app_assistant.rst [hunk 1]
```diff
@@ -160,3 +160,17 @@ This field is appended to the block of chat messages, i.e. attached after the me
...
+Most AI tasks will be run as part of the background job system in Nextcloud which only runs jobs every 5 minutes by default.
+To pick up scheduled jobs faster you can set up background job workers that process AI tasks as soon as they are scheduled:
+
+run the following occ commands a daemon (you can also spawn multiple, for parallel processing):
+
+.. code-block::
+
+   occ background-job:worker 'OC\TaskProcessing\SynchronousBackgroundJob'
+
+Make sure to restart these daemons regularly. For example once a day.
```
Comment (julien-nc) [inline admin_manual/ai/app_assistant.rst:176]: Let's maybe explain why. Something like:
"Apps and server update won't be effective in running daemons, that's why they need to be restarted from time to time."

File: developer_manual/digging_deeper/task_processing.rst [hunk 2]
```diff
@@ -277,17 +296,129 @@ A **Task processing provider** will usually be a class that implements the inter
...
 
-This class would typically be saved into a file in ``lib/TextProcessing`` of your app but you are free to put it elsewhere as long as it's loadable by Nextcloud's :ref:`dependency injection container<dependency-injection>`.
+This class would typically be saved into a file in ``lib/TaskProcessing`` of your app but you are free to put it elsewhere as long as it's loadable by Nextcloud's :ref:`dependency injection container<dependency-injection>`.
+
+Providing supplemental inputs and outputs
```
Comment (julien-nc) [inline developer_manual/digging_deeper/task_processing.rst:343]:
```suggestion
Providing additional inputs and outputs
```
or `optional`?

**Review by julien-nc — CHANGES_REQUESTED** [review]
Yay :rocket:

**Review by julien-nc — APPROVED** [review]
:boom:

**Pull request merged**